docs/File-Formats.md

```diff
@@ -125,6 +125,33 @@ The first four rows of the clinical data file contain tab-delimited metadata abo
 - Row 2: **The attribute Descriptions**: Long(er) description of each clinical attribute
 - Row 3: **The attribute Datatype**: The datatype of each clinical attribute (must be one of:  STRING, NUMBER, BOOLEAN)
 - Row 4: **The attribute Priority**: A number which indicates the importance of each attribute.  In the future, higher priority attributes will appear in more prominent places than lower priority ones on relevant pages (such as the [Study View](http://www.cbioportal.org/study?id=brca_tcga)). A lower number indicates a higher priority.
+    ```
```

Comment:
Please fix in line 127 (above): "A lower number indicates a higher priority."  Should be: "A higher number indicates a higher priority."